hardware/USBtin_MultiblocV8.cpp

```diff
@@ -303,11 +305,10 @@ void USBtin_MultiblocV8::StopThread()
 	ClearingBlocList();
 }
 
-void USBtin_MultiblocV8::ManageThreadV8(bool States, bool debugmode)
+void USBtin_MultiblocV8::ManageThreadV8(bool States, unsigned int debugmode)
```

Comment:
Why did you change this back to unsigned int and using/comparing it as a bool ? It is a boolean